string mode
string side
---
bool success
hrl_ellipsoidal_control/EllipsoidParams e_params
